### PR TITLE
Complete step 1

### PR DESCRIPTION
**Pull request by glasss13**
1. What is the purpose of this PR?
Finish step 1
2. What changes did you make? Why?
I implemented the methods for the `PricingUtil` class and added tests.
3 .What bugs did you find while testing?
I had to implement an empty constructor for the python code to be able to run the constructor.

**This PR Specific:**

1. What challenges did you face while writing the module from scratch?
None
2. How did you ensure your unit tests are comprehensive?
I made sure to include various combinations of different important values for the three inputs to `calcVal`.
3. Did you have enough guidance to complete the task?
It was a little bit ambiguous as to whether `val` should be updated, but I assumed it was.

### CODE REVIEW

File: src/pricingutil.cpp [hunk 1]
```diff
@@ -1 +1,16 @@
 #include "pricingutil.h"
+
+PricingUtil::PricingUtil() {}
```
Comment (minnce) [inline src/pricingutil.cpp]: initialize val here please

**Review by minnce — CHANGES_REQUESTED** [review]

File: src/pricingutil.cpp (hunk 1 above)
Comment (glasss13) [reply src/pricingutil.cpp]: Done.

**Review by minnce — APPROVED** [review]
Thanks for making that change! LGTM :)

**Comment by glasss13** [comment]
> Thanks for making that change! LGTM :)

No problem! Thanks.

**Pull request merged**